criteria/use-plain-english.md

```diff
@@ -20,7 +20,7 @@ Language that is easy to understand lowers the cost of creating and maintaining
 * All codebase documentation MUST be in English.
 * All [source code](../glossary.md#source-code) MUST be in English, except where [policy](../glossary.md#policy) is machine interpreted as code.
 * All bundled policy not available in English MUST have an accompanying summary in English.
-* Any translation MUST be up to date with the English version and vice versa.
+* If the codebase maintainers have committed to providing any translations, then those translation MUST be up to date with the English version and vice versa.
```

Comment:
Let's try to avoid considering English as the unique reference 🙂 There is a list of authoritative, committed languages, that must contain English, but English does not have to be the only reference.

```suggestion
* If the codebase maintainers have committed to providing authoritative any translations, then those translations MUST contain the same content across languages.
```